src/cli/InputParser.ts

```diff
@@ -1,25 +1,25 @@
-import Convert = require("./Index");
 import * as fs from "fs";
-import {defaults} from "lodash";
+import { defaults } from "lodash";
 import * as path from "path";
-import {ApiDoc2InterfaceGroupingMode, ApiDoc2InterfaceParameters} from "../core/ApiDoc2Interface";
-import {BuilderOptions} from "../core/ApiDoc2InterfaceBuilder";
+import { ApiDoc2InterfaceGroupingMode, ApiDoc2InterfaceParameters } from "../core/ApiDoc2Interface";
+import { BuilderOptions } from "../core/ApiDoc2InterfaceBuilder";
+
+import Convert = require("./Index");
 
 type CLIFlags = Record<keyof typeof Convert.flags, any>;
 
 interface ConfigFlags extends BuilderOptions, ApiDoc2InterfaceParameters {}
 
-export class InputParser {
-
+export default class InputParser {
```

Comment:
Don't think linter should decide wether we export default or not
Let's remove this rule or configure it so we can export without default

Reply:
This rule says that in modules with a single export we should use default export, which sounds ok. Maybe you thought it works differently?

Reply:
It sounds ok, but actually https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html
We have removed all default exports in our other projects

Reply:
Maybe we should contact airbnb and tell them about "default is bad"? 🤔

Reply:
No

Reply:
Please fix

Reply:
Done